omero/users/command-line-interface.txt

```diff
@@ -127,20 +127,56 @@ To import a file image.tiff, use::
 
     $ bin/omero import image.tiff
 
-Many options can be passed to the :omerocmd:`import`. They can be listed using
-the :option:`-h` option::
+The options available to the import command are:
 
-    $ bin/omero import -h
+.. option:: -h, --help
 
-The :option:`-f` option displays all the used files for importing then exits::
+    Display the help for the import command
 
-    $ bin/omero import -f image.tif
-    $ bin/omero import -f images_folder
+.. option:: -f
 
-The :option:`--depth` option sets the number of directories to scan down for
-files (default: 4)::
+    Display all the used files for importing then exits::
 
-    $ bin/omero import --depth 1 images_folder
+        $ bin/omero import -f image.tif
+        $ bin/omero import -f images_folder
+
+.. option:: --depth DEPTH
+
+    Set the number of directories to scan down for files (default: 4)::
+
+        $ bin/omero import --depth 1 images_folder
+
+.. option:: --skip [SKIP [SKIP]...]
+
+    Specify optional import steps to skip. Multiple steps can be skipped and
```

Comment:
I think we need to say why you'd want to use this option, ie import speed, maybe others. And maybe wat that is costing you. Especially as this page is aimed at users who may have little idea what these steps are.